### ISSUE TITLE
OverconstrainedError facingMode

### ISSUE THREAD

**Issue by CKESSLER** [zxing-js/library#122]
Since the last update, i'am not able to use my webcam anymore.
I always get the same error : OverconstrainedError {name: "OverconstrainedError", message: null, constraint: "facingMode"}

Everything work fine if i change the function decodeFromInputVideoDevicein BrowserCodeReader.js like this : 
```
constraints = {
    video: { facingMode: { exact: 'environment' } }
};
```
by : 
```
constraints = {
    video: { facingMode: 'environment' }
};
```

**Comment by odahcam** [comment]
I see, because your webcam is not `facingMode: environment`, webcams have `facingMode: user`. I will publish a hotfix for that, thanks.

**Comment by Exilz** [comment]
When will this small fix be released ? :)

**Comment by odahcam** [comment]
Published v0.9.4.
